### ISSUE TITLE
Accessibility: Properties table - tabbable element's role 'rowgroup' is not a widget role

### ISSUE THREAD

**Issue by nmgokhale** [elyra-ai/canvas#1389]
Ref: 3486

Fix accessibility violation in Virtualized table - `The tabbable element's role 'rowgroup' is not a widget role`
![Screenshot 2023-03-07 at 4 09 59 PM](https://user-images.githubusercontent.com/25124000/223585109-fc0e68d6-cb74-49c4-892c-dacb096d2d2d.png)

Element Location - 
```
<div
    style="box-sizing: border-box; direction: ltr; height: 160px; position: relative; width: 643px; will-change: transform; overflow: hidden;"
    tabindex="0"
    role="rowgroup"
    class="ReactVirtualized__Grid ReactVirtualized__Table__Grid"
    aria-label="Table Input">
```
